field/bitmap.go

```diff
@@ -200,6 +200,10 @@ func (f *Bitmap) IsSet(n int) bool {
 		return false
 	}
 
+	if n == 1 && !f.spec.DisableAutoExpand {
```

Comment:
Could you please elaborate on the rationale behind this specific change? I'm having difficulty understanding the problem they're intended to solve.

Reply:
The issue locates on the pack func, when bitmap is auto expanded (meaning that the first bit is active and DisableAutoExpand is false) this will still try to pack the field 1. 
To fix this issue I added this validation, to return false when field 1 is active but not really because of the auto expansion. This condition doesn't contemplates the bits 65, 129 and on, yet.

Reply:
The final decision for this was to not allow composite specs with bitmaps with auto expansion, with this rule this problem is solved for now.